hazelcast/src/main/resources/hazelcast-full-example.yaml

```diff
@@ -1666,8 +1666,7 @@ hazelcast:
       expiry-policy-factory:
         class-name: com.example.cache.MyExpirePolicyFactory
       cache-entry-listeners:
-        cache-entry-listener:
-          old-value-required: false
+        - old-value-required: false
```

Comment:
Why this change?

Reply:
The `cache-entry-listener` key was unnecessary (and I think misleading as well) so I removed it. In commit 6ac6337dbf I added a test verifying that passing the listeners as a YAML sequence works.

It should be possible to configure multiple cache entry listeners for a cache in the config. The value of the `cache-entry-listeners` can be any `YamlCollection` (either a `YamlMapping` or a `YamlSequence`), but since only its values matter during traversal, and the keys are ignored, I think it makes much more sense to put a sequence into the example.

Note: at the first glance it is misleading that [MemberConfigDomProcessor#handleCacheNode() calls # cacheListenerHandle()](https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/internal/config/MemberDomConfigProcessor.java#L1874), which in turn filters children by a [`nodeName = "cache-entry-listener"` predicate](https://github.com/hazelcast/hazelcast/blob/52f37e1cba76b74f33c8668f9212dce42e9ff570/hazelcast/src/main/java/com/hazelcast/internal/config/MemberDomConfigProcessor.java#L2066). But this [#cacheListenerHandle() method is overriden in YamlMemberDomConfigProcessor](https://github.com/hazelcast/hazelcast/blob/52f37e1cba76b74f33c8668f9212dce42e9ff570/hazelcast/src/main/java/com/hazelcast/internal/config/YamlMemberDomConfigProcessor.java#L550) and the filtering is removed (and the nodeNames are ignored here).